prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -1,120 +1,167 @@
+      name: weave-net
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-c1f3803",
+            "original-request": {
+              "url": "/k8s/v1.6/net?v=latest",
+              "date": "Thu Apr 20 2017 09:43:20 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    apiVersion: v1
+    kind: ServiceAccount
+  - metadata:
+      labels:
+        name: weave-net
+      name: weave-net
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-c1f3803",
+            "original-request": {
+              "url": "/k8s/v1.6/net?v=latest",
+              "date": "Thu Apr 20 2017 09:43:20 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
     spec:
-      hostNetwork: true
-      hostPID: true
-      containers:
-        - name: weave
-          image: weaveworks/weave-kube:latest
-          imagePullPolicy: Always
-          command:
-            - /home/weave/launch.sh
-          livenessProbe:
-            initialDelaySeconds: 30
-            httpGet:
-              host: 127.0.0.1
-              path: /status
-              port: 6784
-          securityContext:
-            privileged: true
-          volumeMounts:
+      template:
+        metadata:
+          labels:
+            name: weave-net
+        spec:
+          tolerations:
+            - key: node-role.kubernetes.io/master
+              effect: NoSchedule
+          containers:
+            - name: weave
+              image: 'weaveworks/weave-kube:latest'
+              imagePullPolicy: Always
+              command:
+                - /home/weave/launch.sh
+              env: []
+              livenessProbe:
+                initialDelaySeconds: 30
+                httpGet:
+                  host: 127.0.0.1
+                  path: /status
+                  port: 6784
+              volumeMounts:
+                - name: weavedb
+                  mountPath: /weavedb
+                - name: cni-bin
+                  mountPath: /host/opt
+                - name: cni-bin2
+                  mountPath: /host/home
+                - name: cni-conf
+                  mountPath: /host/etc
+                - name: dbus
+                  mountPath: /host/var/lib/dbus
+                - name: lib-modules
+                  mountPath: /lib/modules
+              resources:
+                requests:
+                  cpu: 10m
+              securityContext:
+                privileged: true
+            - name: weave-npc
+              image: 'weaveworks/weave-npc:latest'
+              imagePullPolicy: Always
+              resources:
+                requests:
+                  cpu: 10m
+              securityContext:
+                privileged: true
+          volumes:
             - name: weavedb
-              mountPath: /weavedb
+              emptyDir: {}
```

Comment:
The diff is quite confusing in this section :(
Do we wanna manually sort the keys for the purpose of the diff?